lib/manageiq/providers/ibm_cloud/cloud_tools/sdk/branch.rb

```diff
@@ -49,6 +49,7 @@ def collection(call_back, **kwargs)
             # @return [Hash] The JSON return of the operation with symbolic key names.
             def request(call_back, **kwargs)
               request = send_request(call_back, **kwargs)
+              return if request.nil?
```

Comment:
NOTE: this is required because not every VPC API call returns a value, specifically DELETE actions explicitly return `nil`